app/mailers/patient_mailer.rb

```diff
@@ -66,7 +66,9 @@ def assessment_sms_weblink(patient)
       url = new_patient_assessment_jurisdiction_lang_initials_url(dependent.submission_token, dependent.jurisdiction.unique_identifier, lang&.to_s,
                                                                   dependent&.initials_age)
       contents = "#{I18n.t('assessments.sms.weblink.intro', locale: lang)} #{dependent&.initials_age('-')}: #{url}"
-      patient.update(last_assessment_reminder_sent: DateTime.now) # Update last send attempt timestamp before Twilio call
+      # Update last send attempt timestamp before Twilio call
+      patient.last_assessment_reminder_sent = DateTime.now
+      patient.save(touch: false)
```

Comment:
Will `touch: false` result in incorrect purge dates as calculated by the `expected_purge_ts` field which uses `updated_at` which util now would have been changing daily when assessments were sent due to the way this code was before?